Can't build without LTO and `-experimental-hermetic-seal-at-link`

```diff
diff --git a/Makefile b/Makefile
index 7c13a24..34dca74 100644
--- a/Makefile
+++ b/Makefile
@@ -4,8 +4,7 @@ TOOLSET := toolset.json
 
 TRIPLE := aarch64-none-none-elf
 SWIFT := swift
-SWIFT_BUILD_FLAGS := --triple $(TRIPLE) --toolset $(TOOLSET) -c release -Xswiftc -Osize \
-                     --experimental-lto-mode=full -Xswiftc -experimental-hermetic-seal-at-link
+SWIFT_BUILD_FLAGS := --triple $(TRIPLE) --toolset $(TOOLSET) -c release
 OBJCOPY := llvm-objcopy
 QEMU := qemu-system-aarch64
 
diff --git a/toolset.json b/toolset.json
index 723a923..92aceff 100644
--- a/toolset.json
+++ b/toolset.json
@@ -16,7 +16,6 @@
             "-nostdlib",
             "-static",
             "--gc-sections",
-            "--print-gc-sections",
             "--strip-all",
             "--orphan-handling=error",
             "-T", "linker.ld",
```

```console
$ make
swift build --triple aarch64-none-none-elf --toolset toolset.json -c release
Building for production...
error: link command failed with exit code 1 (use -v to see invocation)
clang: warning: argument unused during compilation: '-F/Applications/Xcode-16.3.0.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-F/Applications/Xcode-16.3.0.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/PrivateFrameworks' [-Wunused-command-line-argument]
ld.lld: error: undefined symbol: memcpy
>>> referenced by <compiler-generated>:0 (/<compiler-generated>:0)
>>>               /Users/kebo/swift_os/.build/aarch64-none-none-elf/release/Kernel.build/Kernel.swift.o:($e6KernelAAV4mainyyFZTf4d_n)                                                          

ld.lld: error: /Users/kebo/swift_os/.build/aarch64-none-none-elf/release/Font.build/font.swift.o:(.swift_modhash) is being placed in '.swift_modhash'
ld.lld: error: /Users/kebo/swift_os/.build/aarch64-none-none-elf/release/Kernel.build/Kernel.swift.o:(.swift_modhash) is being placed in '.swift_modhash'
ld.lld: error: /Users/kebo/swift_os/.build/aarch64-none-none-elf/release/RaspberryPi.build/Framebuffer.swift.o:(.swift_modhash) is being placed in '.swift_modhash'
ld.lld: error: /Users/kebo/swift_os/.build/aarch64-none-none-elf/release/RaspberryPi.build/mbox.swift.o:(.swift_modhash) is being placed in '.swift_modhash'
ld.lld: error: /Users/kebo/swift_os/.build/aarch64-none-none-elf/release/RaspberryPi.build/putchar.swift.o:(.swift_modhash) is being placed in '.swift_modhash'
ld.lld: error: /Users/kebo/swift_os/.build/aarch64-none-none-elf/release/RaspberryPi.build/uart.swift.o:(.swift_modhash) is being placed in '.swift_modhash'
clang: error: ld.lld command failed with exit code 1 (use -v to see invocation)
[10/11] Linking Kernel
make: *** [swift] Error 1
```

--- a/Package.swift
+++ b/Package.swift
@@ -24,6 +24,7 @@ let package = Package(
         .executableTarget(
             name: "Kernel",
             dependencies: [
+                "Support",
                 "AsmSupport",
                 .byName(name: "RaspberryPi", condition: .when(traits: ["RASPI"])),
             ],
@@ -37,6 +38,7 @@ let package = Package(
             ],
         ),
         .target(name: "Font", swiftSettings: swiftSettings),
+        .target(name: "Support", swiftSettings: swiftSettings),
         .target(name: "AsmSupport"),
     ],
 )

--- a/Sources/Support/memcpy.swift
+++ b/Sources/Support/memcpy.swift
@@ -1,0 +1,15 @@
+@_silgen_name("memcpy")
+func memcpy(
+    _ dst: UnsafeMutableRawPointer,
+    _ src: UnsafeRawPointer,
+    _ n: Int,
+) -> UnsafeMutableRawPointer {
+    let dst = unsafe dst.bindMemory(to: UInt8.self, capacity: n)
+    var dstSpan = unsafe MutableSpan(_unsafeStart: dst, count: n)
+    let src = unsafe src.bindMemory(to: UInt8.self, capacity: n)
+    let srcSpan = unsafe Span(_unsafeStart: src, count: n)
+    for i in dstSpan.indices {
+        dstSpan[i] = srcSpan[i]
+    }
+    return .init(dst)
+}

--- a/Sources/Support/memset.swift
+++ b/Sources/Support/memset.swift
@@ -1,0 +1,13 @@
+@_silgen_name("memset")
+func memset(
+    _ dst: UnsafeMutableRawPointer,
+    _ val: CInt,
+    _ len: Int,
+) -> UnsafeMutableRawPointer {
+    let dst = unsafe dst.bindMemory(to: UInt8.self, capacity: len)
+    var span = unsafe MutableSpan(_unsafeStart: dst, count: len)
+    for i in span.indices {
+        span[i] = UInt8(truncatingIfNeeded: val)
+    }
+    return .init(dst)
+}

